apps/photos/src/services/clipService.ts

```diff
@@ -166,7 +170,7 @@ class ClipServiceImpl {
                 return;
             }
             const localFiles = getPersonalFiles(await getAllLocalFiles(), user);
-            const existingEmbeddings = await getLatestClipImageEmbeddings();
+            const existingEmbeddings = await getLocalEmbeddings(model);
```

Comment:
Have updated to `getLocalEmbeddings`, because we anyways are blocking the scheduling of clip embedding extraction on remote sync to finish. So, local embedding will be the latest already.